lib/mastodon/search_cli.rb

```diff
@@ -6,17 +6,28 @@
 
 module Mastodon
   class SearchCLI < Thor
+    option :parallel, default: 2, aliases: [:p]
     desc 'deploy', 'Create or update an ElasticSearch index and populate it'
     long_desc <<~LONG_DESC
       If ElasticSearch is empty, this command will create the necessary indices
       and then import data from the database into those indices.
 
       This command will also upgrade indices if the underlying schema has been
       changed since the last run.
+
+      With the --parallel option, Execute tasks in parallel. The default is 2,
+      which specifies the number of processors. If 'auto' is specified,
+      it is automatically derived.
     LONG_DESC
     def deploy
-      processed = Chewy::RakeHelper.upgrade
+      processed = Chewy::RakeHelper.upgrade parallel: parallel
       Chewy::RakeHelper.sync(except: processed)
     end
+
+    private
+
+    def parallel
+      options[:parallel] == 'auto' ? true : Integer(options[:parallel], exception: false) || 1
```

Comment:
If parallel=1, doesn't that mean it's not parallel at all?

Reply:
Maybe Chewy is smart enough to handle that case though...

Reply:
1 if an invalid value other than 'auto' or a number is specified.
I think `false` is appropriate, so change it.

Reply:
Chewy may accept nil. Should we remove false?

Reply:
It's probably fine to use false. However, the `Chewy::RakeHelper.sync(except: processed)` method is still not being passed `parallel` param.